examples/almondbread.ml

```diff
@@ -24,7 +24,7 @@ let mandelbrot x y =
       A.rgb_888 ~b:(f (-.pi2_3)) ~g:(f 0.) ~r:(f pi2_3)
 
 let xlate dx dy f x y = f (x -. dx) (y -. dy)
-let scale k f = let k1 = 1./.k in fun x y -> f (x *. k1) (y *. k1)
+let[@warning "-32"] scale k f = let k1 = 1./.k in fun x y -> f (x *. k1) (y *. k1)
```

Comment:
I would suggest commenting this out instead. It's unused and really there for a "future improvement". There is a similar precedent above.